model/edition.py

```diff
@@ -344,26 +344,25 @@ def sort_key(edition):
                 # lowest priority.
                 return -100
 
-            if source == self.data_source:
+            if source == license_source:
                 # This Edition contains information from the same data
                 # source as the LicensePool itself. Put it below any
                 # Edition from one of the data sources in
                 # PRESENTATION_EDITION_PRIORITY, but above all other
                 # Editions.
                 return -1
 
+            elif source.name == DataSourceConstants.METADATA_WRANGLER:
+                # The metadata wrangler is slightly less trustworthy
+                # than the license source, for everything except cover
+                # image (which is handled by
+                # Representation.quality_as_thumbnail_image)
+                return -1.5
+
             if source.name in DataSourceConstants.PRESENTATION_EDITION_PRIORITY:
-                id_type = edition.primary_identifier.type
-                if (id_type == Identifier.ISBN and
-                    source.name == DataSourceConstants.METADATA_WRANGLER):
-                    # This ISBN edition was pieced together from OCLC data.
-                    # To avoid overwriting better author and title data from
-                    # the license source, rank this edition lower.
-                    return -1.5
```

Comment:
It turns out we _always_ want to rank the metadata wrangler lower than the license source.